ini/stm32f1.ini

```diff
@@ -157,6 +157,7 @@ board_build.variant  = MARLIN_F103Rx
 board_build.ldscript = ldscript.ld
 extra_scripts        = ${common.extra_scripts}
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
+  pre:buildroot/share/PlatformIO/scripts/stm32_serialbuffer.py
```

Comment:
This is really needed? If so, I suspect that it should be added in the common stm32 env, for every stm32 board.

Reply:
This is #21177 ported to STM32. Some G2/G3 commands with line number and checksum can exceed 64 bytes and be corrupted. I think this issue only happens with hardware serial (Robin E3 and Creality 4.2.2 confirmed), USB serial may be unaffected.

Reply:
So I think it should be added in the stm32_common and apply to all stm32 boards.